File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cs
using System;

public class Solution {
    public int[] solution(string[] wallpaper) {
        int top = int.MaxValue;
        int left = int.MaxValue;
        int down = int.MinValue;
        int right = int.MinValue;
        
        for(int i = 0; i < wallpaper.Length; i++)
        {
            for(int j = 0; j < wallpaper[0].Length; j++)
            {
                if(wallpaper[i][j] == '#')
                {
                    top = Math.Min(top, i);
                    left = Math.Min(left, j);
                    down = Math.Max(down, i + 1);
                    right = Math.Max(right, j + 1);
                }
            }
        }
        return new int[] {top, left, down, right};
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    int minRow = 1e9, minCol = 1e9;
    int maxRow = 0, maxCol = 0;
    
    for(int i = 0; i < wallpaper.size(); i++)
    {
        for(int j = 0; j < wallpaper[0].size(); j++)
        {
            if(wallpaper[i][j] == '#')
            {
                minRow = min(minRow, i);
                minCol = min(minCol, j);
                maxRow = max(maxRow, i);
                maxCol = max(maxCol, j);
            }
        }
    }
    return {minRow, minCol, maxRow + 1, maxCol + 1};
}